lib/galaxy/jobs/__init__.py

```diff
@@ -764,7 +764,30 @@ def convert_legacy_destinations(self, job_runners):
                     log.warning("Legacy destination with id '%s' could not be converted: Unknown runner plugin: %s" % (id, destination.runner))
 
 
-class JobWrapper( object ):
+class HasResourceParameters:
+
+    def get_resource_parameters( self, job=None ):
+        # Find the dymically inserted resource parameters and give them
+        # to rule.
+
+        if job is None:
+            job = self.get_job()
+
+        app = self.app
+        param_values = job.get_param_values( app, ignore_errors=True )
+        resource_params = {}
+        try:
+            resource_params_raw = param_values[ "__job_resource" ]
+            if resource_params_raw[ "__job_resource__select" ].lower() in [ "1", "yes", "true" ]:
```

Comment:
I don't want a dependency between galaxy.web.framework.helpers and galaxy.jobs. I guess we don't want to expand the web stuff to allow ``1`` as well - we should be moving toward JSON booleans in the API - not strings for web stuff. So I would say we keep it like it is for now? If you really want me to modify it - I can switch this out for a generic tool handling boolean thing I guess - but I'd prefer not to.